changelog.d/20230418_183334_30907815+rjmello_bugfix_display_name.rst

```diff
@@ -0,0 +1,7 @@
+Bug Fixes
+^^^^^^^^^
+
+- When attempting to start an endpoint while running both a legacy `funcx-endpoint`
```

Comment:
This fixes a bug between alpha versions.  So IMO it might confuse the user more than it informs them on how to fix an issue.  But I suppose it doesn't hurt.

Reply:
Yeah, good point. I'll remove it.